src/coins.h

```diff
@@ -416,13 +416,14 @@ class CCoinsViewCache : public CCoinsViewBacked
     void AddCoin(const COutPoint& outpoint, Coin&& coin, bool possible_overwrite);
 
     /**
-     * Emplace a coin into cacheCoins without performing any checks, marking
-     * the emplaced coin as dirty.
+     * Emplace a coin into cacheCoins without performing any checks, optionally
+     * marking the emplaced coin as dirty.
```

Comment:
I'm not sure that's the best though, since we do not mark a coin as not dirty. That is the default state.

What about
"marking the coin as dirty unless `set_dirty` is set to false"?

Reply:
That sounds good to me :+1:

Reply:
Done.